includes/Modules/Analytics.php

```diff
@@ -1368,6 +1393,15 @@ protected function create_analytics_site_data_request( array $args = array() ) {
 		$request->setIncludeEmptyRows( true );
 		$request->setViewId( $profile_id );
 
+		$dimension_filter_clauses = array();
+		if ( ! empty( $args['dimension_filters'] ) ) {
+			$dimension_filters       = $args['dimension_filters'];
+			$dimension_filter_clause = new Google_Service_AnalyticsReporting_DimensionFilterClause();
+			$dimension_filter_clause->setFilters( array( $dimension_filters ) );
```

Comment:
I believe this is incorrect? `$dimension_filters` should already be passed as the array required here, so it must not be wrapped again.

Reply:
Updated.